ideascube/mediacenter/forms.py

```diff
@@ -17,6 +18,9 @@ class Meta:
         widgets = {
             'lang': LangSelect,
         }
+        labels = {
+            'original': _('Source file (maximum size : 500 Mo)'),
```

Comment:
Sneaked the limit in there, didn't you? 😆 

Shouldn't the limit instead be in a help text?

And why not change the `verbose_name` on the model instead?